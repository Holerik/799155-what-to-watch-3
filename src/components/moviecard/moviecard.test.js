// moviecard.test.js
import React from 'react';
import renderer from 'react-test-renderer';
import MovieCard from './moviecard.jsx';

const filmInfo = {
  id: 201,
  title: `Rampage`,
  poster: `img/rampage.jpg`,
  altPoster: `rampage poster`,
};

const movieCardActivateHandler = () => {};
const movieCardClickHandler = () => {};

it(`<MovieCard /> should render Rampage small card`, () => {
  const tree = renderer
    .create(<MovieCard
      movie={filmInfo}
      movieCardActivateHandler={movieCardActivateHandler}
      movieCardClickHandler={movieCardClickHandler}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
