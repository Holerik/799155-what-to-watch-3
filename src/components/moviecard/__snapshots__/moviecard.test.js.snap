// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieCard /> should render moviee small card 1`] = `
<article
  className="small-movie-card catalog__movies-card"
>
  <div
    className="small-movie-card__image"
    id="201"
    onClick={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
  >
    <img
      alt="rampage poster"
      height="175"
      src="img/rampage.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
    onClick={[Function]}
  >
    <a
      className="small-movie-card__link"
      href="#"
    >
      Rampage
    </a>
  </h3>
</article>
`;
